qemu/tests/virtio_fs_support_win_fs.py

```diff
@@ -0,0 +1,78 @@
+    5) Copy a program like 7z-x64.exe and autoit script to the shared dir.
+    6) Create a new user.
+    7) login a common user and install 7z-x64.exe by autoit script.
+    8) After test, restore the image.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    fs_target = params.get("fs_target")
+    package_path = params.get("executable_package_path")
+    autoIt_path = params.get("autoIt_path")
+    script_path = params.get("script_path")
+    reg_add_username = params.get("reg_add_username")
+    reg_add_pwd = params.get("reg_add_pwd")
+
+    vm = env.get_vm(params.get("main_vm"))
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    error_context.context("Create the viofs service", test.log.info)
+    virtio_fs_utils.create_viofs_service(test, params, session)
+    error_context.context("add the viofs service", test.log.info)
+    reg_add_cmd = params.get("reg_add_cmd")
+    session.cmd(reg_add_cmd)
+    error_context.context("stop the viofs service", test.log.info)
+    virtio_fs_utils.stop_viofs_service(test, params, session)
+    error_context.context("start the viofs service", test.log.info)
+    virtio_fs_utils.start_viofs_service(test, params, session)
+
+    winutils_driver_letter = utils_misc.get_winutils_vol(session)
+    shared_driver_letter = virtio_fs_utils.get_virtiofs_driver_letter(test,
+                                                                      fs_target,
+                                                                      session)
+    winutils_pack_path = package_path % winutils_driver_letter
+    autoIt_path = autoIt_path % winutils_driver_letter
+    script_path = script_path % winutils_driver_letter
+    copy_cmd = "xcopy %s %s:\\ /Y" % (winutils_pack_path, shared_driver_letter)
+    error_context.context("Copy the installation package to shared dir",
+                          test.log.info)
+    session.cmd(copy_cmd)
+
+    error_context.context("Create the new user", test.log.info)
+    session.cmd(params.get("add_user_cmd"))
+    error_context.context("Replace the default username and password",
+                          test.log.info)
+    session.cmd(reg_add_username)
+    session.cmd(reg_add_pwd)
+    session = vm.reboot(session)
+    error_context.context("Open the shared directory for autoit",
+                          test.log.info)
+    session.cmd("explorer %s:" % shared_driver_letter)
+    time.sleep(3)
+    result_on_shared = "%s:\\result.txt" % shared_driver_letter
+    error_context.context("Run autoit script...", test.log.info)
+    session.cmd("start /w " + autoIt_path + " " + script_path + " " +
+                result_on_shared)
+    output = session.cmd_output("type " + result_on_shared)
+    error_context.context("The title of window detected is %s" % output,
+                          test.log.info)
+    if "Network Error" in output:
```

Comment:
Could we also get the other error info? 
if not PASS:

Reply:
> Could we also get the other error info? if not PASS:

Do you want show the details about error at here?  @xiagao